src/entry_node.cpp

```diff
@@ -162,4 +164,25 @@ Status EntryNode::deserialize(const tensorflow::TensorProto& proto, InferenceEng
     }
     return StatusCode::OK;
 }
+
+Status EntryNode::createShardedBlob(InferenceEngine::Blob::Ptr& dividedBlob, const InferenceEngine::TensorDesc& dividedBlobDesc, InferenceEngine::Blob::Ptr blob, size_t i, size_t step, const NodeSessionMetadata& metadata, const std::string blobName) {
+    // if condition is perf optimization
```

Comment:
This comment line is not needed since you moved it to entry node - whole method is perf optimization